main.js

```diff
@@ -0,0 +1,109 @@
+const $purchaseButton = document.getElementsByClassName("purchase-button")[0];
+const $purchasemoney=document.getElementsByClassName("purchase-money")[0];
+const $lottolist= document.getElementsByClassName("lotto-tickets-container")[0];
+const $resultbutton=document.getElementsByClassName("result")[0];
+const $modal=document.getElementsByClassName("modal")[0];
+const $modalclosex=document.getElementsByClassName("modal-close-x")[0];
+const $restartbutton=document.getElementsByClassName("restart")[0];
+const $winninglottos = document.querySelectorAll('.winning input');
+const $bonuswinninglotto = document.querySelector('.bonus input');
+const $resulttable= document.querySelector('.result-table');
+const $profit = document.querySelector('.profit');
+$modal.style.display = 'none';
+
+let lottos=[]                   //구입로또리스트
```

Comment:
이거 포함해서 아래에 세미콜론이 빠져있는 곳이 꽤 있네요..!

```suggestion
let lottos=[];                   //구입로또리스트
```